roles/openshift_metrics/README.md

```diff
@@ -5,37 +5,65 @@ OpenShift Metrics Installation
 
 Requirements
 ------------
-It requires subdomain fqdn to be set.
-If persistence is enabled, then it also requires NFS.
+
+The following variables need to be set and will be validated:
+
+- `metrics_hostname`: hostname used on the hawkular metrics route.
+
+- `metrics_project`: project (i.e. namespace) where the components will be
+  deployed.
+
 
 Role Variables
 --------------
 
-From this role:
+For default values, see `defaults/main.yaml`.
```

Comment:
I think we should describe the defaults here... I realize it's a duplicate, that or make this a link to defaults/main.yaml